src/lib/schema.ts

```diff
@@ -56,8 +56,20 @@ interface OnnxFile extends SingleFile {
     type: 'onnx';
 }
 
+export interface DiscordData {
```

Comment:
This interface isn't used in any `data/*` file, so it shouldn't be here in `schema.ts`.